### PR TITLE
Don't log org.eclipse.jetty.io.EofException as warning

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java [hunk 1]
```diff
@@ -122,7 +123,11 @@ private void honourMaxKeepAliveRequests() {
             boolean reportedError = false;
 
             if (error != null) {
-                if (!(error instanceof OverloadException || error instanceof BindingNotFoundException)) {
+                if (error instanceof EofException) {
+                    log.log(Level.FINE,
```
Comment (gjoranv) [inline jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java]: Why so many line breaks?

Comment (gjoranv) [reply jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java]: I also suggest to static import FINE and WARNING

Comment (bjorncs) [reply jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java]: Because I follow the line limit strictly.

Comment (gjoranv) [reply jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/HttpRequestDispatch.java]: At which column?

**Pull request merged**